packages/adp-tooling/test/unit/writer/__snapshots__/index.test.ts.snap

```diff
@@ -80,18 +80,18 @@ server:
             url: http://sap.example
           ignoreCertErrors: false
     - name: fiori-tools-proxy
-      afterMiddleware: fiori-tools-preview
+      afterMiddleware: compression
       configuration:
-        ignoreCertErrors: false
+        ignoreCertError: false # If set to true, certificate errors will be ignored. E.g. self-signed certificates will be accepted
```

Comment:
When did this comment get added: `# If set to true, certificate errors will be ignored. E.g. self-signed certificates will be accepted`? I think there were no comments in the ui5.yaml previously

Reply:
That comment was always generated for a Fiori app project but not for adp. By reusing the existing app code, it is now added here as well.